Programming Assignment/Week 1/RajShah_Week2/costFunction.m

```diff
@@ -1,36 +0,0 @@
-function [J, grad] = costFunction(theta, X, y)
```

Comment:
Why did you delete these files? Add all the files where you have written a code except for the warmup one

Reply:
By mistake I had uploaded assignment for week-2 in week-1's folder so i deleted that.